crates/router/src/connector/cashtocode/transformers.rs

```diff
@@ -288,12 +299,26 @@ impl<F, T>
 }
 
 #[derive(Debug, Deserialize)]
-pub struct CashtocodeErrorResponse {
+#[serde(untagged)]
+pub enum CashtocodeErrorResponse {
+    StringError(CashtocodeStringErrorResponse),
+    IntError(CashtocodeIntErrorResponse),
+}
+
+#[derive(Debug, Deserialize)]
+pub struct CashtocodeStringErrorResponse {
     pub error: String,
     pub error_description: String,
     pub errors: Option<Vec<CashtocodeErrors>>,
 }
 
+#[derive(Debug, Deserialize)]
+pub struct CashtocodeIntErrorResponse {
```

Comment:
There is no need to define new struct to handle a field which can have multiple type. Instead define `error` as serde_json::Value